src/Markdig/Extensions/MediaLinks/MediaOptions.cs

```diff
@@ -90,6 +91,8 @@ public MediaOptions()
 
         public string Height { get; set; }
 
+        public bool Controls { get; set; }
```

Comment:
How about something more explicit, e.g. `AddControlsProperty`?

Reply:
Well you could make the same argument about `width` and `height`, both of them are optional.  
`Controls` is the name of the HTML attribute and I think `Controls` true or false will be easily understood, no need to be explicit

Reply:
@MihaZupan  I don't think it's needed myself but I have renamed the property name as you suggested